NAS-101923 doc "Replication Performance" field appearance condition (#1319)
freenas/freenas-docs: userguide/intro.rst

* The :guilabel:`Name` field has been added to
  :menuselection:`Tasks --> Replication Tasks --> Add`.

> * The :guilabel:`Replication Performance` in

Reviewer: ```suggestion
* :guilabel:`Replication Performance` in
```

  :menuselection:`Tasks --> Replication Tasks --> ADD`

>   has changed to only appear when *SSH* is chosen for the

Reviewer 2: ```suggestion
  now only appears when *SSH* is chosen for the
```
